frontend/src/components/pages/Scheduling/SelectDateTime.tsx

```diff
@@ -249,6 +249,7 @@ const SelectDateTime = ({
     setRecurringEndDate(selectedDateObj);
 
     const recurringDate = new Date(selectedDateObj);
+    recurringDate.setHours(23,59,59,999); // Need to set it to EOD so that recurring schedules are created properly in the backend
```

Comment:
let's use this? (https://date-fns.org/v2.28.0/docs/endOfDay)
```suggestion
    const recurringDate = endOfDay(new Date(selectedDateObj))
```